agrirouter-api-java-api/src/main/java/com/dke/data/agrirouter/api/service/messaging/EndpointsListService.java

```diff
@@ -0,0 +1,12 @@
+package com.dke.data.agrirouter.api.service.messaging;
+
+import agrirouter.request.payload.account.Endpoints;
+import com.dke.data.agrirouter.api.dto.onboard.OnboardingResponse;
+import com.dke.data.agrirouter.api.service.parameters.EndpointsListParameters;
+
+public interface EndpointsListService extends MessagingService<EndpointsListParameters> {
```

Comment:
I would rather talk about a `ListEndpointsService` which has has methods like `requestAllFiltered` and `requestAll` than the current solution.

Reply:
Reasonable input; Class renamed. Function however called requestFullList because requestAll is in my view not understandable